docs/source/start-software.rst

```diff
@@ -284,14 +304,15 @@ USACE users:
 
 * USERNAME: User H7
```

Comment:
H7 is district specific.

Reply:
`Oracle User` and `Oracle Password` are probably the best. still a bit vague but most staff would either understand that or at least be near, or have access directly to, someone who does.